crates/ruff/resources/test/fixtures/pydocstyle/D401.py

```diff
@@ -2,6 +2,8 @@
 
 from functools import cached_property
 
+from gi.repository import GObject
+
```

Comment:
Is such third-party import in tests okay?

Reply:
Yeah totally fine. We don't actually run the code, only analyze it.